maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm

```diff
@@ -30036,6 +30036,9 @@
 /area/fiorina/station/power_ring)
 "sgb" = (
 /obj/item/ammo_box/magazine/misc/flares/empty,
+/obj/effect/landmark/nightmare{
```

Comment:
- You're copy pasting a massive piece/the entire LV CLF ship insert onto Fiorina. It's current location is not very good and remember ship hull/walls can not be broken/torn down meaning there is no flanking routes or different ways to enter it than the south side.
- Same as before, use Noop turf/area when there are no changes between a small area on the nightmare insert and base map.
- Why are there so many invis walls up North on the blocked off part? If you're worried about burrowers etc. check the area code for the space there, see if they can burrow as i believe they already cant.
- Tldr. Re-do it in its entirety or drop this change.